content/en/templates/data-templates.md

```diff
@@ -114,6 +114,12 @@ You can use the following code to render the `Short Description` in your layout:
 
 Note the use of the [`markdownify` template function][markdownify]. This will send the description through the Markdown rendering engine.
 
+## Use data in front matter
+
+You might want to access different properties of your data in front matter.
+For example, you want to specify a `location` for a page with specific information about that location.
+
+To do so, use the [`index` function][index].
```

Comment:
That someone was me, thank you for providing the solution, @CollierCZ !
I will take the liberty of suggesting a change more specific to what the root cause of my problem was:
```suggestion
### Named values in variables

You might want to access the key name as a variable. To do so, use the [`index` function][index], with the variable as the second parameter.

{{< highlight plaintext >}}
{{ $keyName := "Name" }}
{{ index .Site.Data.User0123 $keyName }}
{{< /highlight >}}

If the `Name` key was an object, you could also access its properties:

{{< highlight plaintext >}}
{{ $keyName := "Name" }}
{{ (index .Site.Data.User0123 $keyName).propertyName }}
{{< /highlight >}}
```

I had to suggest the highlight shortcode here, using three backticks seems to collide with Github's suggestions.

Reply:
You can get around that on GitHub by using four backticks for the suggestion itself

Reply:
````suggestion
## Named values in variables

You might want to access a key name as a variable.
To do so, use the [`index` function][index] with the variable as the second parameter.
```plaintext
{{ $keyName := "Name" }}
{{ index .Site.Data.User0123 $keyName }}
```

If the `Name` key is an object, you can also access its properties:

```plaintext
{{ $keyName := "Name" }}
{{ (index .Site.Data.User0123 $keyName).propertyName }}
```
````